### DISCUSSION TITLE
Disable modded ore gen

### DISCUSSION THREAD

**Discussion by ghost** [EwyBoy/OreTweaker#105] [Help Wanted!] [answered]
Hi i am using version 1.18.2-3.4.1 is it possible to use this to disable generation of modded ores? I am using some mods that generate ore with no config to turn them off. I am overhauling ore gen for my pack so i need a way to disable it

**Comment by Bebobobo** [comment, marked as answer]
yes. If you look here on discord you can see the correct syntax. Feel free to message in discord if you have issues.

https://discord.com/channels/305535757441826817/390890978980003840/1192544525290324059
